cmd/agent/app/integrations.go

```diff
@@ -612,40 +606,16 @@ func getVersionFromReqLine(integration string, lines string) (*integrationVersio
 		return nil, fmt.Errorf("Found several matches for %s version in %s\nAborting", integration, lines)
 	}
 
-	version, err := parseVersion(groups[0][1])
+	version, err := semver.NewVersion(groups[0][1])
 	if err != nil {
 		return nil, err
 	}
 	return version, nil
 }
 
-func pipCheck() error {
```

Comment:
Good catch! Did we forget to remove this deprecated function?

Reply:
Yeah, after using the downloader it stopped being used